lib/sikuli/common.sikuli/common.py

```diff
@@ -125,12 +125,26 @@ def _wait_for_loaded(self, component, similarity=0.70):
         @param component: The waiting component
         @param similarity: The similarity of component. Default: 0.70.
         """
+        is_exists = False
         for counter in range(50):
```

Comment:
That's great idea.

However, there is a bad news that, the wait time `N` will take more than `N` seconds.
The reason is each image will wait `0.1` sec, see following code:
```python
             for pic, _, _ in component:
                  if exists(Pattern(pic).similar(similarity), 0.1):
                      break
```
So, when we have more than 1 image in the component, it will take more than wait time `N`.
I'll plan to do the overall code change in #679.